src/app/components/report-problem/report-problem.component.ts

```diff
@@ -13,7 +13,10 @@ import { fields } from "./report-problem.schema.json";
     <baw-wip>
       <baw-form
         title="Report Problem"
-        [subTitle]="subTitle"
+        subTitle="
+          Complete the form below to report a problem.
+          Alternatively, we have a <a href='https://github.com/QutEcoacoustics/baw-server/issues'>Github Issues</a> page.
```

Comment:
do not hardcode url for repo.

Add it to evironment if you really need it.

I'd suggest  not linking to the repo at all.

Reply:
Ill turn this into a separate PR